seacrowd/sea_datasets/baybayin/baybayin.py

```diff
@@ -0,0 +1,187 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+from PIL import Image
```

Comment:
LGTM 🙏! 

Except could we mention the newly used `pillow` library in the `requirements.txt`?

Wdyt about this? @holylovenia @akhdanfadh

Reply:
Ah yes! Let me add a simple wrapper to check if the module exists, similar to [this](https://github.com/SEACrowd/seacrowd-datahub/pull/556#discussion_r1586359470) discussion.